app/controllers/reviews_controller.rb

```diff
@@ -16,6 +16,40 @@ def create
     end
   end
 
+  def upvote
```

Comment:
you should create separate resources for your upvotes and downvotes to maintain RESTful conventions

Consider posting to url's like:

```
/reviews/349/upvotes
/reviews/432/downvotes
```